doc/api/access-codes.md

```diff
@@ -0,0 +1,57 @@
+Access Code Pairs
+=================
+
+After you have successfully logged into your [account], you are able to create access code pairs.  These code pairs are allowed to [tokenize and detokenize](tokens.md) data associated with your account.
+
+
+Creation of Code Pairs
+----------------------
+
+When you fetched your [account resource][account] and discovered the [link relations](links.md) to other services, there was one titled `account-accessCode`.  It has an associated profile, so let's fetch that first.
```

Comment:
There are two spaces between the sentences here, which is inconsistent with account.md (or account.md is inconsistent with this).